### PR TITLE
Document reuse identifiers convention

### PR DESCRIPTION
**Pull request by mouhcine**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,10 @@ To use YLTableView, you should subclass `YLTableViewDataSource`. After implement
 @end
 ```
 
+#### Reuse identifiers and Swift
+YLTableView relies on using a cell class name as its reuse identifier. To achieve this, you should use `NSStringFromClass([YLExampleTextCell class])` even with Swift code. Using `String(YLExampleTextCell)` can cause issues with YLTableView.
```
Comment (benasher44) [inline README.md]: nit: "…as that cell's reuse identifer.…"

Also, can you meantion that you can also make ObjC use a sane name by marking your UITableViewCell's class as `@objc(MyCellSubclass)` (that just re-declares the same name)?

**Comment by SSheldon** [comment]
@mouhcine, I poked around a bit and found that we don't really need to require this convention and it's pretty simple to remove it. I've done that as part of another change in #21, if that's received favorably I don't think we'll need to update the readme :) I'll update depending on the feedback for #21.

File: README.md [hunk 2]
```diff
@@ -74,6 +74,11 @@ To use YLTableView, you should subclass `YLTableViewDataSource`. After implement
 @end
 ```
 
+#### Reuse identifiers and Swift
+YLTableView relies on using a cell class name as that cell's reuse identifier. To achieve this, you should use `NSStringFromClass([YLExampleTextCell class])` even with Swift code (`NSStringFromClass(ExampleTextCell)`). Using `String(YLExampleTextCell)` can cause issues with YLTableView.
+It is recommended to mark classes as `@objc(YLExampleTextCell) class ExampleTextCell: UITableViewCell` to make `NSStringFromClass` generate a user-readable name (i.e. `YLExampleTextCell`).
```
Comment (benasher44) [inline README.md]: nit: "…mark cell classes like `@objc`…"

Last nit I promise, and then I think this is ready :)

**Comment by benasher44** [comment]
Going to close this now that #22 is merged and fixes this issue and makes this no longer a convention.

**Pull request closed without merging**